CODING_STYLE.md

```diff
@@ -0,0 +1,39 @@
+# Coding Style
```

Comment:
I suggest tomove this file to `/docs/` dir. It's a good practice (in OS as well as DEV) to have the root as clean as possible for maintanability.

I'd like to move the CODE_OF_CONDUCT and CONTRIBUTING there as well, but I'm not sure if it is possible due to external references.